features/bootstrap/MfaContext.php

```diff
@@ -1,58 +1,23 @@
 <?php
-namespace Sil\SspMfa\Behat\context;
```

Comment:
It looks like we completely lost the namespace here. Was that intentional?

Reply:
I did this according to my understanding of the Behat documentation.

Reply:
Looking back at the [docs](https://behat.org/en/latest/user_guide/context.html#context-class-requirements) it doesn't explicitly say to omit the namespace, but it does mention an autoloader defined for the bootstrap directory.